### PR TITLE
Filter out test publisher

### PR DESCRIPTION
**Pull request by kentsanggds**
## What 

Filter out test publisher to prevent it from being harvested and pushed to the data.gov.uk Find frontend. 

Also update the buildpack for deployment as we upgraded to Ruby 2.6.6

## Reference 

https://trello.com/c/bi425MX0/135-block-data-from-test-publisher-on-ckan-from-reaching-dgu-find

### CODE REVIEW

File: config/environments/staging.rb [hunk 1]
```diff
@@ -3,4 +3,5 @@
 Rails.application.configure do
   config.assets.compile = true
   config.ckan_v26_base_url = "https://ckan.staging.publishing.service.gov.uk"
+  config.test_publisher = ""
```
[the remark this answers is not in the document]
Comment (kentsanggds) [reply config/environments/staging.rb:6]: We want to block out test data from reaching Find on production but its ok if test data reaches staging Find, we can then test things out on the test publisher.

File: spec/features/ckan_v26_package_sync_spec.rb [hunk 2]
```diff
@@ -100,4 +101,10 @@
     expect(Dataset.all).to_not include dataset_to_delete
     expect { get_from_es(dataset_to_delete.uuid) }.to raise_error(/404/)
   end
+
+  it "rejects data from the test publisher specified in the config" do
+    subject.perform
+    datasets = Dataset.all
+    expect(datasets.map(&:uuid)).to_not include dataset_from_test_publisher_id
```
[the remark this answers is not in the document]
Comment (kentsanggds) [reply spec/features/ckan_v26_package_sync_spec.rb:108]: ok, will try that

[the remark this answers is not in the document]
Comment (1pretz1) [reply spec/features/ckan_v26_package_sync_spec.rb:108]: To add to my comment you could reduce this by a line by doing ` expect(Dataset.pluck(:uuid)).to_not include dataset_from_test_publisher_id`

[the remark this answers is not in the document]
Comment (kentsanggds) [reply spec/features/ckan_v26_package_sync_spec.rb]: ah right, will try that out now

**Pull request merged**